catalog/dags/common/constants.py

```diff
@@ -31,5 +31,6 @@
 
 POSTGRES_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
 OPENLEDGER_API_CONN_ID = os.getenv("OPENLEDGER_API_CONN_ID", "postgres_openledger_api")
+API_CONN_ID = os.getenv("API_CONN_ID", "api")
 AWS_CONN_ID = os.getenv("AWS_CONN_ID", "aws_conn_id")
 AWS_RDS_CONN_ID = os.environ.get("AWS_RDS_CONN_ID", AWS_CONN_ID)
```

Comment:
Just making a note here for myself - I'm going to make an issue to remove the `conn_id` configuration - we really don't need to allow the connection ID itself to be configurable, we should pick something and stick with it 😅 This PR follows the paradigm though so no need to change here!

Reply:
Issue made! https://github.com/WordPress/openverse/issues/2294

Reply:
Totally agree! 😅